classes/plugins/Plugin.inc.php

```diff
@@ -512,7 +512,9 @@ function installData($hookName, $args) {
 
 		// Install all data files.
 		foreach($dataFiles as $dataFile) {
-			$sql = $installer->dataXMLParser->parseData($dataFile);
+			$dataXMLParser = new DBDataXMLParser();
```

Comment:
I would have proposed this line and the one after that [515-516] to go above the `foreach`, I don't think that it is needed to be instantiated for each `dataFile`

Reply:
Yes, thanks! But let me review your other comment before I do that.

Reply:
The other comment obsoleted this one.